content/news/releases/prestashop-8-1-1-maintenance-release.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  PrestaShop 8.1.1 is available
+subtitle: First patch for branch 8.1.x is available
+date: 2023-08-07
+authors: [ PrestaShop ]
+image: /assets/images/2023/07/811_banner_1200x627.png
+opengraph_image: /assets/images/2023/07/811_banner_1200x627.png
+twitter_image: /assets/images/2023/07/811_banner_1024x512.png
+icon: icon-leaf
+tags:
+- security
+- version
+- patch
+- releases
+- "8.1"
+- "8.1.x"
+---
+
+The first patch for PrestaShop 8.1 is available. This release contains a few security fixes, so we highly recommend you to upgrade your shop as soon as possible.
+
+![8.1.1 is available!](/assets/images/2023/07/811_banner_1534x424.png)
+
+Version 8.1.1 fixes a few security issues found by security researchers and solves [a great number of bugs](https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Amerged+milestone%3A8.1.1+label%3A%22Bug+fix%22+-label%3A%22E2E+Tests%22+) reported by the community and project's Quality Assurance team.
+
+You can go to the [release page](https://github.com/PrestaShop/PrestaShop/releases/tag/8.1.1) to see a full changelog, but it's worth mentioning a few notable bug fixes:
+
+- [Check if product is active and orderable during checkout](https://github.com/PrestaShop/PrestaShop/pull/33156)
+- [Fix for product editor not storing some fields in multi shop context](https://github.com/PrestaShop/PrestaShop/pull/33151)
+- [Register commonly used deprecated modifiers to Smarty config to avoid warnings in back office](https://github.com/PrestaShop/PrestaShop/pull/33136)
+- [Fix for child themes/multishop themes translations](https://github.com/PrestaShop/PrestaShop/pull/33133)
+- [Fix for issue with PDF files being always in English if generated from back office](https://github.com/PrestaShop/PrestaShop/issues/32530)
+
+Some of those issues were reported on multiple occasions, so we are happy to finally have them fixed. We would like to thank everyone who reported those issues and helped us to fix them.
+
+For more details regarding security fixes, you can check the [security advisories](#security-advisories) section below.
+
+It is recommended to upgrade your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before. You can upgrade to the latest version using the [1-Click Upgrade module](https://github.com/PrestaShop/autoupgrade/releases/).
```

Comment:
```suggestion
It is recommended to upgrade your shop quickly in order to benefit from these fixes. Of course, don’t forget to backup before. You can upgrade from 1.7 versions to the latest version using the [1-Click Upgrade module](https://github.com/PrestaShop/autoupgrade/releases/).
```

Reply:
I wouldn't specify it, you can upgrade from all versions?

Reply:
Yes, I can understand your reasoning @MatShir, but I think the message is quite clear as it is now.

Reply:
@Hlavtox you can with CLI, with the module you can't really. There are some issues.
No worries, I wanted to avoid disappointment and suggest for interested 1.6 users to look for other options